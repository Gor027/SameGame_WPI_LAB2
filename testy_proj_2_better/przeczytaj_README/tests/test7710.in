6666y 6 yy6y6 yy   y666y 66  y 6 yy6yy 66 y66  6yyyyyy y 66yyyyy6 yy6y
6 666 666y y 6y
yy666 yy 66y
66y66y6 yy6 yyy66y  y6666y66y y   6 6yyy yy6 yy 6y6yy6
yy  6yy6y66yyy  y6   y6yy  66   y  yy6    6yy  6   yy66 
 6yy  6 y 6yy6yyy  6y 6y y yy 
6 y
 y y   666yy66yyyy6 6yy 66 6  6y
6 6 y66yyyyy6  y6
6y  y yyy6y yyy66 y 6 y 66  yy6   y6
 y 6 yy6  y 6y6 y 
 yy6 y yy6y 66y66  y yyyy6 6yy y6y6 66666  y 6y 66   yy6y
y  y6  yyyyyy 6y6   y  6y
y6666y  yy6y   66y 6  6yyy 6y
