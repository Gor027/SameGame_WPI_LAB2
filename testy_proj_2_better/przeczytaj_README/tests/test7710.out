          6
        yy6
        6y6 6   6
    y 66y66yy yyy
6   6 6yyy6yy6y6y     6
6  66 y6y6yyyyyyy     6y66
y6 66y6666yy6yyy6   6 66yy 6y
6y666yyyyyyy66yy6   y66y6y 66
y666y66yy6yyy6y6yy yy6yy66 yy     y yy     6         y
6y66yy66y6yy6y6666 66yyyy6 6yyy6  666y 6  y6    yy yy6
66666yyyyyyy6yyy6y6y6yy6y666yy6yyyyyy6yy6 yy6yy66yyyy66y
66666yyyyy6yy6666yy6yy6yyy66yy6y6y6666666y6yy6y6666yyyy6y66yyyyy6yy6y
